### ISSUE TITLE
How to actually run script?

### ISSUE THREAD

**Issue by CptanPanic** [yang/greminders2todoist#1]
Basic question,
The Readme says "Run the script ..." but doesn't actually say how to run, or what to run? Can you give some more specifics? I did the the previous 2 steps.

Thanks.

**Comment by yang** [comment]
@CptanPanic Sure thing, I've updated the README - in short, the `pip install...` line installs a bin script called `greminders2todoist`, so now you should be able to just run `greminders2todoist` (it doesn't take any args).  Hope that helps, feel free to ask if you run into further issues!

**Comment by CptanPanic** [comment]
Sorry for keep having issues, but I finally got it running, but now have issue where I am running on a system that doesn't have a desktop, and links does pop up asking me to login for goauth on todoist, but it doesn't seem to work in lynks. Any ideas?

**Comment by yang** [comment]
No worries at all.

I'm looking into how to expose the URL you should navigate to.

**Comment by yang** [comment]
Ah, luckily the URL to go to is already getting printed at the console.  Do you see something like the following?

```
Please visit this URL to authorize this application: https://todoist.com/oauth/authorize?response_type=code&client_id=XXXXXXXXXXXXXXXXXXX&redirect_uri=http%3A%2F%2Flocalhost%3A3423%2F&scope=data%3Aread_write&state=XXXXXXXXXXXXXXXXXXX&access_type=offline
```

**Comment by yang** [comment]
Also, are you ssh-ing into this system?  If so, you would need to ssh into it with this extra `-L` argument:

```
ssh -L3423:localhost:3423 you@host
```

That tunnels any connections to localhost:3423 to the remote host's port 3423.  So after you go to the printed URL in your browser and authorize the app in Todoist, the browser will redirect you to localhost:3423, so this -L argument ensures that will work and that the console app will get passed the auth data.

Let me know if that works for you.

**Comment by CptanPanic** [comment]
Yes but if I use this url on another desktop, it tries to redirect to
localhost. If it change the todoist API to redirect to another machine it
will have to be https.

**Comment by CptanPanic** [comment]
Actually the ash tunnel is a good idea and will try it. Thanks.

**Comment by yang** [comment]
Were you able to get this working for you?

**Comment by CptanPanic** [comment]
Yes the ssh tunnel worked. Thanks
